src/utils.jl

```diff
@@ -214,3 +214,40 @@ macro matchable(expr)
         Base.length(x::$name) = $(length(fields) + 1)
     end |> esc
 end
+
+# Check for linear coefficients
+coefficients(expr,sym::SymbolicUtils.Sym) coefficients(expr,[sym])
+function coefficients(expr,syms::AbstractArray{SymbolicUtils.Sym})
+    res = Dict{SymbolicUtils.Sym,Bool}
```

Comment:
It should be `Dict{Any,Bool}`, we shouldn't just restrict this to Sym. We can have `coefficient(a*sin(x), sin(x)) => a`